### DISCUSSION TITLE
pyrfc support statement

### DISCUSSION THREAD

**Discussion by chandra-a** [SAP/PyRFC#297] [Q&A] [answered]
Dear All,

can you please help with this
is pyrfc officially supported by SAP ?
if not - if i were to develop a business critical process based on pyrfc - and  i get asked "how would this application be supported in case of issues" - what can i respond

i see this is being used by a broad community and so i am not worried in general about support or finding solutions
its just that is there any "formal support statement" that i can share with my business for them to approve this in their critical business process

Thanks
Chandrakanth

**Comment by bsrdjan** [comment, marked as answer]
Hello Chandrakanth,

PyRFC is SAP open source and as such not covered by enterprise support model and services. As per [PyRFC/LICENSE 7](https://github.com/SAP/PyRFC/blob/main/LICENSE#L55), the usage is at own risk and responsibility. It may sound risky to users familiar with (commercial) enterprise support and fixed service level agreements but that is the common model for open source in general.

PyRFC support with fixed SLA may be required for different reasons and can provided as (paid) consulting service. It is one of optional commercial services, on top of PyRFC free usage: 

- Users' training
- Consulting how to use PyRFC in your project
- Help with implementing custom business, technical or security features or integrations
- Support with fixed service level agreements (SLA)
- Any other help to meet critical deadlines or solve complex challenges, in projects with PyRFC

If interested in these services, you can create PyRFC issue or question like this and then we can follow-up with SAP partner or customer management.

Hope this helps,
Srdjan
